### DISCUSSION TITLE
DON'T DEAL WITH @CAN-KAT CS STEALER

### DISCUSSION THREAD

**Discussion by Tgibby001** [community/community#62071] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

HE KEEPS DELETING MY MESSAGES AND I WILL KEEP WRITING MORE REVIEWS ABOUT HOW YOU SCAMMED ME. HE TOOK MONEY FOR ME FOR OVER 2 MONTHS AND HE HAS BEEN IGNORING ME. DON'T DEAL WITH HIM, HE WILL SCAM YOU. REFUND MY MONEY.

HE KEEPS DELETING OUR CHATS ON TELEGRAM AND MY COMMENTS ON HERE. PAY ME BACK MY MONEY BRO. HE SCAMMED OTHER PEOPLE TOO. 

HE RUNS THIS GITHUB @CAN-KAT (CS STEALER)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Tgibby001 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
